### PR TITLE
feat: Add handler which can allow SQL timeout without stream processing beeing halted

### PR DESCRIPTION
**Pull request by janlesage**

### CODE REVIEW

File: src/RoadRegistry.BackOffice/Messages/AnnounceRoadNetworkExtractDownloadTimeoutOccurred.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+namespace RoadRegistry.BackOffice.Messages;
+
+using Be.Vlaanderen.Basisregisters.EventHandling;
+
+public class AnnounceRoadNetworkExtractDownloadTimeoutOccurred : IMessage
+{
+    public string RequestId { get; set; }
+
+    //TODO: - Extend all road network events with a RoadNetworkRevision
```
Comment (rikdepeuter) [inline src/RoadRegistry.BackOffice/Messages/AnnounceRoadNetworkExtractDownloadTimeoutOccurred.cs:9]: pending TODO?

File: src/RoadRegistry.BackOffice/Messages/RoadNetworkExtractGotRequestedV2.cs [hunk 2]
```diff
@@ -5,12 +5,21 @@ namespace RoadRegistry.BackOffice.Messages;
...
     public string ExternalRequestId { get; set; }
     public string RequestId { get; set; }
     public string When { get; set; }
-}
\ No newline at end of file
+}
+
+public interface IRoadNetworkExtractGotRequestedMessage : IMessage
+{
+    public RoadNetworkExtractGeometry Contour { get; set; }
```
Comment (rikdepeuter) [inline src/RoadRegistry.BackOffice/Messages/RoadNetworkExtractGotRequestedV2.cs:20]: `public` accessor is not relevant for interfaces

Comment (janlesage) [reply src/RoadRegistry.BackOffice/Messages/RoadNetworkExtractGotRequestedV2.cs]: Moved into own file in Core and removed accessor

**Review by rikdepeuter — APPROVED** [review]

**Pull request merged**